packages/trader/src/Modules/Trading/Components/Form/WheelPicker/wheel-picker.scss

```diff
@@ -0,0 +1,83 @@
+.wheel-picker {
+    background: var(--general-main-1);
+    width: 100%;
+    height: 7.6rem;
+    padding: 0.8rem;
+    margin-top: 0.8rem;
+    position: relative;
+    display: flex;
+    &__wheel {
+        text-align: center;
+        width: 100%;
+        height: 6.8rem;
+        &__placeholder {
+            height: 1.8rem;
+        }
+    }
+    &__actions {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        &__btn {
+            padding: 0rem 0.8rem;
+            min-width: 1.6rem;
+            background: var(--general-section-1);
+            :disabled {
+                .chevron-icon svg {
+                    stroke: #f2f3f4;
+                }
+            }
+        }
+        &--chevron-up {
+            transform: rotate(180deg);
+        }
+    }
+}
+
+.wheel-picker-mobile {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    gap: 0.1rem;
+    height: 13rem;
+    text-align: center;
+    margin: 0;
+    position: relative;
+    .picker-selected-wrapper {
```

Comment:
nit-pick:
it can be nested under  `.picker-selected {` `as &-wrapper {`,
and all classnames starting with `.picker` can be placed under `.picker {` :)